src/main/blackbox/blackbox_io.h

```diff
@@ -72,8 +72,10 @@ bool blackboxDeviceFlushForce(void);
 bool blackboxDeviceOpen(void);
 void blackboxDeviceClose(void);
 
+#ifdef USE_FLASHFS
 void blackboxEraseAll(void);
 bool isBlackboxErased(void);
+#endif
```

Comment:
Fair enough. I was going for 'absolute correctness', but as you say, it does not have a parctical advantage, and readability is better without it. Fixed.